dydx_user/views.py

```diff
@@ -10,13 +11,32 @@
 
 class User(GenericViewSet):
     def position_id(self, request):
-
         result = {"message": None, "error": None}
         admin = DydxAdmin()
         try:
             position_id = admin.get_position_id()
             result["message"] = {"position_id": position_id}
             return Response(result, status.HTTP_200_OK)
+        except DydxApiError or ValueError as e:
+            e = vars(e)
+
+            result["error"] = e["msg"]["errors"][0]["msg"]
+            return Response(result, status.HTTP_400_BAD_REQUEST)
         except Exception as e:
             result["error"] = str(e)
             return Response(result, status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+    def register_user(self,request):
+        result = {"message": None, "error": None}
+        admin = DydxAdmin()
+        try:
+            user = admin.register_user()
+            if user is None:raise Exception("Fail to register")
```

Comment:
if we fail to register a user, wouldn't we get some sort of an error msg from dydx that'll be thrown as an exception from `admin.register_user()` ? 

Also, not a good practise to raise exception inside the try block itself.
Raise that exception in `admin.register_user()` if you think a user could be failed to registered without a known reason.

Reply:
Hnn right will change it

Reply:
We get some error and that catches in exception